src/pages/sunday-school-page.tsx

```diff
@@ -1,7 +1,35 @@
 export const SundaySchool = () => {
   return (
     <div>
-      <h1>Sunday School</h1>
-    </div>
+      <body>
+        <div id="SundaySchoolSummary">
+          <h1 id="SundaySchoolHeader">Sunday School</h1>
+          <p>Let's learn Arabic, Islamic studies, and Quran at Sunday School</p>
+        </div>
+        <div id="TeacherInfoSection">
+          <h2 id="TeacherInfoHeader">Teacher Information</h2>
+          <img id="TeacherImage" src="teacherimage.jpg"/>
```

Comment:
Images of the teacher going to be a thing? I took a stab at it assuming it might be but am not quite sure what information to account for here